static/app/components/replays/replayPlayer.tsx

```diff
@@ -80,14 +83,31 @@ function BasePlayerRoot({className}: Props) {
 // Center the viewEl inside the windowEl.
 // This is useful when the window is inside a container that has large fixed
 // dimensions, like when in fullscreen mode.
-const SizingWindow = styled('div')`
+const SizingWindow = styled('div')<{minHeight: number}>`
   width: 100%;
   height: 100%;
+  min-height: ${p => p.minHeight}px !important;
```

Comment:
this could become `min-height Infinitypx !important;`. Any `number` type could be... but it's more likely here because `Infinity` is the default. I'll fix.